givitsite/templates/coordinator.html

```diff
@@ -0,0 +1,144 @@
+{%extends 'base.html'%}
+
+{%block content%}
+<head>
+    <script>
```

Comment:
I'd generally avoid this kind of scripting trick so early in development and let things that are logically separate screens have their own views and templates on the server side. That way all the logic is in a single language and easier to test.

Reply:
I really narrowed down th JS logic to the necessities + moved the functions to a util js class under static folder